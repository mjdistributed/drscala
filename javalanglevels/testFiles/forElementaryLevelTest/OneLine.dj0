class OneLine extends Maggie { int stuff; double method(char c) { return stuff; } }