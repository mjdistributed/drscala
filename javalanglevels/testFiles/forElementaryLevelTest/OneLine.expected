// Language Level Converter line number map: dj*->java. Entries: 1
//     1->3    
class OneLine extends Maggie { private final int stuff; public double method(final char c) { return stuff; } 
  /** This method is automatically generated by the Language Level Converter. */
  public OneLine(int super_i, int super_j, int stuff) {
    super(super_i, super_j);
    this.stuff = stuff;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int stuff() {
    return stuff;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public java.lang.String toString() {
    return getClass().getName() + "(" + 
        stuff() + ", " + 
        i() + ", " + 
        j() + 
        ")";
  }

  /** This method is automatically generated by the Language Level Converter. */
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    else if ((o == null) || (! o.getClass().equals(getClass()))) {
      return false;
    }
    else {
      OneLine cast = ((OneLine) o);
      return (stuff() == cast.stuff()) && 
          (i() == cast.i()) && 
          (j() == cast.j());
    }
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int hashCode() {
    return getClass().hashCode() ^ 
        stuff() ^ 
        i() ^ 
        j();
  }
}
